lib/utils.js

```
@@ -3,9 +3,21 @@
 var bind = require('./helpers/bind');
 var isBuffer = require('is-buffer');
 
+/*global toString:true*/
+
 // utils is a library of generic helper functions non-specific to axios
 
-var _toString = Object.prototype.toString;
+var toString = Object.prototype.toString;
```

Biome (2.1.2)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)